File: beecrowd/1001/js/solution.js
// 1001 - Extremely Basic
// https://judge.beecrowd.com/en/problems/view/1001
//
// Topics: input/output, arithmetic

function solve(input) {
  const [a, b] = input.trim().split(/\s+/).map(Number);

  return `X = ${a + b}\n`;
}

if (require.main === module) {
  const fs = require('fs');
  const input = fs.readFileSync(0, 'utf8');

  process.stdout.write(solve(input));
}

module.exports = { solve };

File: beecrowd/1001/cpp/solution.cpp
// 1001 - Extremely Basic
// https://judge.beecrowd.com/en/problems/view/1001
//
// Topics: input/output, arithmetic

#include <iostream>

int sum(int a, int b) {
    return a + b;
}

#ifndef UNIT_TEST
int main() {
    int a;
    int b;

    std::cin >> a >> b;
    std::cout << "X = " << sum(a, b) << '\n';

    return 0;
}
#endif
